docs/source/guides/user/chapters/plugins.rst

```diff
@@ -267,3 +267,28 @@ this is not set.
 
 Finally, any failures in the Pre/Post scripts will not alter the
 status of the corresponding jobs.
+
+Tests' logs plugin
+~~~~~~~~~~~~~~~~~~
+
+It's natural that Avocado will be used in environments where access to
+the integral job results won't be easily accessible.
+
+For instance, on Continuous Integration (CI) services, one usually
+gets access to the output produced on the console, while access to
+other files produced (generally called artifacts) may or may not be
+acessible.
+
+For this reason, it may be helpful to simply output the logs for tests
+that have "interesting" outcomes, which usually means that fail and
+need to be investigated.
+
+To show the content for test that are canceled, skipped and fail, you
+can set on your configuration file::
+
+  [job.run.testlogs]
```

Comment:
Looking at this namespace and key, it is not so clear to me. I had to take a look on this documentation.

maybe `job.run.outputlogs` or `job.output.testlogs`?

Reply:
Yeah, I had a hard time coming up with a namespace, and I think it's still not good enough.  I think I like your `job.output.testlogs` better, by a tiny little bit.  I'll send a v2 changing it.